### DISCUSSION TITLE
linut mint, debian, improvements & ideas

### DISCUSSION THREAD

**Discussion by birtisen** [linuxmint/discussions#162] [Ideas]
as requested on another git thread, 
few things to see & check, about LMDE5, hope it would be changed/updated in the LMDE6..
[ideas-lmdeB.pdf](https://github.com/linuxmint/discussions/files/12643573/ideas-lmdeB.pdf)

**Comment by AtomicRobotMan0101** [comment]
I believe the admins would like this to be individual posts. That way each concept/idea can be implemented/address individually.
